pp-mock/services/data/mock_data.go

```diff
@@ -119,11 +119,38 @@ func init() {
 			},
 		},
 	}
+
+	// TripBasicV3[2,0]
 	TripsBasicV3[2].Segments[0].Departure.Location = &transportv3.TransitEventLocation{
 		Location: &transportv3.TransitEventLocation_LocationCode{
 			LocationCode: &typesv2.LocationCode{
-				Code: "DUS",
-				Type: 2,
+				Code: "LAN",
```

Comment:
How comes that the departure location of the first segment is "LAN" here in the source code and "DUS" in the mock data?

Reply:
Changed the mock data to match the source code.